libs/redis/patches/010-fix-makefile.patch

```diff
@@ -0,0 +1,26 @@
+Index: redis-5.0.5/src/Makefile
+===================================================================
+--- redis-5.0.5.orig/src/Makefile
++++ redis-5.0.5/src/Makefile
+@@ -197,7 +197,6 @@ persist-settings: distclean
+ 	echo REDIS_LDFLAGS=$(REDIS_LDFLAGS) >> .make-settings
+ 	echo PREV_FINAL_CFLAGS=$(FINAL_CFLAGS) >> .make-settings
+ 	echo PREV_FINAL_LDFLAGS=$(FINAL_LDFLAGS) >> .make-settings
+-	-(cd ../deps && $(MAKE) $(DEPENDENCY_TARGETS))
+ 
+ .PHONY: persist-settings
+ 
+@@ -248,12 +247,10 @@ dict-benchmark: dict.c zmalloc.c sds.c s
+ 	$(REDIS_CC) -c $<
+ 
+ clean:
+-	rm -rf $(REDIS_SERVER_NAME) $(REDIS_SENTINEL_NAME) $(REDIS_CLI_NAME) $(REDIS_BENCHMARK_NAME) $(REDIS_CHECK_RDB_NAME) $(REDIS_CHECK_AOF_NAME) *.o *.gcda *.gcno *.gcov redis.info lcov-html Makefile.dep dict-benchmark
+-
++	$(info "prevent dist clean")
```

Comment:
Not sure about this patch.  It seems previous pr #3875 also do not need it.  Anyway I will give it a try later.

Reply:
Removed